dags/common/utils.py

```diff
@@ -291,6 +290,58 @@ def article_type_filter(article_type: str, types: list) -> bool:
         return False
     return article_type.lower() in set(t.lower() for t in types)
 
+def title_filter(case_report: dict) -> bool:
```

Comment:
There is already another title filter. Python doesn't support overloaded functions. I think the existing title function is more flexible.